assets/templates/partials/json-ld/base.tmpl

```diff
@@ -2,7 +2,7 @@
 <script type="application/ld+json">
     {
         "@context": "http://schema.org",
-        "name": {{ .Metadata.Title -}},
+        "name": {{ .Metadata.Title }},
         "publisher": {
         "@type": "Organization",
```

Comment:
@rav-pradhan Small scope creep, but could we please change this to:

```suggestion
        "@type": "GovernmentOrganization",
```

Reply:
Updated